suzieq/db/parquet/parquetdb.py

```diff
@@ -609,7 +613,7 @@ def _get_cp_dataset(self, table_name: str, need_sqvers: bool,
         if filelist:
             return ds.dataset(filelist, format='parquet', partitioning='hive')
         else:
-            return []
+            return None
```

Comment:
Why None instead of empty list?

Reply:
Because we return a dataset as result not a list, why should we return an empty list if normally don't return a list?